### ISSUE TITLE
IPLD Playground

### ISSUE THREAD

**Issue by victorb** [ipld/website#12]
To make people understand the value of IPLD and have something visual on the website, we should have a small visual demo we can place on the top of the landing page.

One idea is to have something similar to http://json-ld.org/playground but a lot lighter and smaller feature set.

Another is to have some sort of visualization that can demonstrate IPLD without interactivity.

Any ideas?

Todo:
- [ ] Come up with demo idea
- [ ] Implement

**Comment by victorb** [comment]
- [ ] Start thinking and come up with some prototype ideas for the playground

**Comment by jbenet** [comment]
Should i spend some time brainstorming here?

**Comment by victorb** [comment]
@jbenet yes! That would be helpful, @nicola should log some of his thoughts as well :+1: 

I know that @dignifiedquire did some work previously as well, at least on the design side, so if you have something to add, please do :)

**Comment by victorb** [comment]
- [x] @nicola will share his sketches and ideas of how it would work

**Comment by nicola** [comment]
See https://github.com/ipld/website/issues/24

**Comment by victorb** [comment]
Nicola mentions the interactive example above. Thought I should clarify the difference between the "playground" and the "interactive example".

Playground -> Usable tool for people who work with IPLD, comes with bunch of resolvers and helps you reason about the data structure you're creating. Meant for people who know the basics about IPLD, and need to quickly demonstrate/do something.

Interactive Example -> Very visual and basic description of how IPLD works and what it does. Meant for people to quickly understand what IPLD really is.
